src/_guides/language/effective-dart/usage.md

```diff
@@ -116,11 +113,18 @@ under `test` that other libraries in `test` import.
 But you can't "cross the streams". A library outside of `lib` should never use a
 relative import to reach a library under `lib`, or vice versa. Doing so will
 break Dart's ability to correctly tell if two library URIs refer to the same
-library. Follow these two rules:
+library, which can lead to unexpected duplicated types.
+
+Follow these two rules:
 
 * An import path should never contain `/lib/`.
 * A library under `lib` should never use `../` to escape the `lib` directory.
 
+In summary, there is no profound reason to prefer the former—it's just shorter-
+but using a single style of imports within `lib` is required to avoid unexpected
+type duplication.
```

Comment:
The summary is a little confusing because the first half seems to disagree with the latter half. It's really talking about two separate concepts: imports that reach in/out of "lib" and imports that don't. I'd just leave this sentence out.

Reply:
Done!